### PR TITLE
Artifactory-cli-go Version 1.2.0

### PR DESCRIPTION
**Pull request by jainishshah17**
Artifactory CLI provides a command line interface for uploading and downloading artifacts to and from Artifactory.

### CODE REVIEW

File: Library/Formula/artifactory-cli-go.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class ArtifactoryCliGo < Formula
+    desc "Artifactory CLI provides a command line interface for uploading and downloading artifacts to and from Artifactory."
+    homepage "https://github.com/JFrogDev/artifactory-cli-go"
+    url "https://bintray.com/artifact/download/jfrog/artifactory-cli-go/1.2.0/artifactory-cli-mac-386/art"
```
Comment (mistydemeo) [inline Library/Formula/artifactory-cli-go.rb]: Can this be built from source instead? We don't accept prebuilt binaries in this repository.

File: Library/Formula/artifactory-cli-go.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    sha256 "79f3084291a4d54cfbf72dc1bbd16e14097077aa52897ed9b4e6fcf448758776"
+    def install
```
Comment (mikemcquaid) [inline Library/Formula/artifactory-cli-go.rb]: Add a newline above this line.

File: Library/Formula/artifactory-cli-go.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+        bin.install "art"
+        bin.install_symlink libexec/"bin/art"
+    end
+    test do
```
Comment (mikemcquaid) [inline Library/Formula/artifactory-cli-go.rb]: Add a newline above this line.

File: Library/Formula/artifactory-cli-go.rb [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+        system bin/"art", "--version"
```
Comment (mikemcquaid) [inline Library/Formula/artifactory-cli-go.rb]: Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

File: Library/Formula/artifactory-cli-go.rb [hunk 5]
```diff
@@ -0,0 +1,13 @@
+class ArtifactoryCliGo < Formula
+    desc "Artifactory CLI provides a command line interface for uploading and downloading artifacts to and from Artifactory."
```
Comment (mikemcquaid) [inline Library/Formula/artifactory-cli-go.rb]: Use 2 character rather than 4 character indents.

**Comment by DomT4** [comment]
Closing in favour of the newly opened https://github.com/Homebrew/homebrew/pull/46891.

**Pull request closed without merging**